MPU9250.h

```diff
@@ -157,6 +164,11 @@ class MPU9250_ {
         b_ahrs = b;
     }
 
+    void set_mag_time_out(const int milliseconds){
+        mag_time_out = milliseconds;
+    }
+    
+
```

Comment:
No need to add this function. Please see 2a151e0eec30f26838af99829f634d260e4b6e21